.github/ISSUE_TEMPLATE/update_supporting_kubernetes.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Update supporting Kubernetes
+about: Dependencies relating to Kubernetes version upgrades
+title: 'Update supporting Kubernetes'
+labels: 'update kubernetes'
+assignees: ''
+
+---
+
+## Update Procedure
+
+- Read [this document](https://github.com/topolvm/pie/blob/main/docs/maintenance.md).
+
+## Before Check List
+
+There is a check list to confirm depending libraries or tools are released.
+
+### Must Update Dependencies
+
+Must update Kubernetes with each new version of Kubernetes.
+
+- [ ] k8s.io/api
+  - https://github.com/kubernetes/api/tags
+    - The supported Kubernetes version is written in the description of each tag.
+- [ ] k8s.io/apimachinery
+  - https://github.com/kubernetes/apimachinery/tags
+    - The supported Kubernetes version is written in the description of each tag.
+- [ ] k8s.io/client-go
+  - https://github.com/kubernetes/client-go/tags
+    - The supported Kubernetes version is written in the description of each tag.
+- [ ] sigs.k8s.io/controller-runtime
+  - https://github.com/kubernetes-sigs/controller-runtime/releases
+- [ ] sigs.k8s.io/controller-tools
+  - https://github.com/kubernetes-sigs/controller-tools/releases
+- [ ] kind
+  - https://github.com/kubernetes-sigs/kind/releases
+
```

Comment:
```suggestion
- [ ] kindest
  - https://hub.docker.com/r/kindest/node/tags
  
```

Reply:
@llamerada-jp 
Do you mean that we should check kindest separately? I assume it is written on the kind release note.

Reply:
The (hash) version of kindest/node is noted in Release notes of kind, so we don't need to check kindest/node release explicitly.